spec/lib/coach/request_serializer_spec.rb

```diff
@@ -34,5 +35,39 @@
         expect(sanitized).to eq('value')
       end
     end
+
+    context "as an instance" do
+      let(:mock_request) do
+        instance_double("Request", :request, format: nil,
+                                             remote_ip:           nil,
+                                             uuid:                nil,
+                                             method:              nil,
+                                             filtered_parameters: nil,
+                                             filtered_env:        {
+                                               "foo" => "bar",
+                                               "HTTP_bar" => "bar"
+                                             })
+      end
+
+      subject(:request_serializer) { described_class.new(mock_request) }
+
+      describe "#serialize" do
+        it "serializes a request" do
+          allow(mock_request).to receive(:fullpath) { raise }
```

Comment:
As far as I can see, this and the spec at line 63 do exactly the same thing. @rthbound have I missed something? The code appears to be identical.

I would lean toward removing the first, while keeping the second as it's testing a more unusual circumstance.